test/functional/feature_pruning.py

```diff
@@ -147,7 +147,7 @@ def reorg_test(self):
 
         # Reboot node1 to clear those giant tx's from mempool
         self.stop_node(1)
-        self.start_node(1, extra_args=["-maxreceivebuffer=20000","-blockmaxsize=5000", "-checkblocks=5", "-disablesafemode"])
```

Comment:
This should also probably set blockmaxweight=20000.

The stop_node / start_node here is an attempt to flush the mempool; but I wonder if that actually does any good if the mempool is being persisted to disk anyway?